R/data-process.R

```diff
@@ -1,229 +1,273 @@
+    dplyr::group_by(data_weighted_burden_wide$simID) |>
+    dplyr::mutate(data_weighted_burden_wide$cumdiff_daly <- cumsum(data_weighted_burden_wide$diff_daly),
+                  data_weighted_burden_wide$cumdiff_yll <- cumsum(data_weighted_burden_wide$diff_yll),
+                  data_weighted_burden_wide$cumdiff_yld <- cumsum(data_weighted_burden_wide$diff_yld))
+
+  data_weighted_burden_wide_collapse <- data_weighted_burden_wide |>
+    dplyr::group_by(data_weighted_burden_wide$time) |>
+    dplyr::summarise(data_weighted_burden_wide$diff_daly_mean <- mean(data_weighted_burden_wide$diff_daly),
+                     data_weighted_burden_wide$diff_daly_min <- min(data_weighted_burden_wide$diff_daly),
+                     data_weighted_burden_wide$diff_daly_max <- max(data_weighted_burden_wide$diff_daly),
+                     data_weighted_burden_wide$diff_yll_mean <- mean(data_weighted_burden_wide$diff_yll),
+                     data_weighted_burden_wide$diff_yll_min <- min(data_weighted_burden_wide$diff_yll),
+                     data_weighted_burden_wide$diff_yll_max <- max(data_weighted_burden_wide$diff_yll),
+                     data_weighted_burden_wide$diff_yld_mean <- mean(data_weighted_burden_wide$diff_yld),
+                     data_weighted_burden_wide$diff_yld_min <- min(data_weighted_burden_wide$diff_yld),
+                     data_weighted_burden_wide$diff_yld_max <- max(data_weighted_burden_wide$diff_yld),
+                     data_weighted_burden_wide$cumdiff_daly_mean <- mean(data_weighted_burden_wide$cumdiff_daly),
+                     data_weighted_burden_wide$cumdiff_daly_min <- min(data_weighted_burden_wide$cumdiff_daly),
+                     data_weighted_burden_wide$cumdiff_daly_max <- max(data_weighted_burden_wide$cumdiff_daly),
+                     data_weighted_burden_wide$cumdiff_yll_mean <- mean(data_weighted_burden_wide$cumdiff_yll),
+                     data_weighted_burden_wide$cumdiff_yll_min <- min(data_weighted_burden_wide$cumdiff_yll),
+                     data_weighted_burden_wide$cumdiff_yll_max <- max(data_weighted_burden_wide$cumdiff_yll),
+                     data_weighted_burden_wide$cumdiff_yld_mean <- mean(data_weighted_burden_wide$cumdiff_yld),
+                     data_weighted_burden_wide$cumdiff_yld_min <- min(data_weighted_burden_wide$cumdiff_yld),
+                     data_weighted_burden_wide$cumdiff_yld_max <- max(data_weighted_burden_wide$cumdiff_yld))
+
+  return(data_weighted_burden_wide_collapse)
+}
+
+#' Perform data smoothing
+#'
+#' This function performs data smoothing.
+#'
+#' @param data_weighted_burden A data frame containing weighted values for burden of disease.
+#' @return A data frame with spline smoothing applied for burden of disease.
+#' @export
+gen_data_weighted_burden_spline <- function(data_weighted_burden) {
+
+  ## Only keep those 0 or negative values
+
+  ## Notes: Delete years 27,30,32-33 for ps3-low; Delete years 2028 for ps4-low
```

Comment:
Can I amend this note to 
## Notes for India project: Delete years 27,30,32-33 for ps3-low; Delete years 2028 for ps4-low?

Reply:
Hi @jzhu20, yes I have made the change to this note